modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -0,0 +1,251 @@
+| "rel_uniqueness_constraint"  | "RELATIONSHIP_PROPERTY_UNIQUENESS"
+
+2+d|Rows: 2
+|===
+
+======
+
+If a database has specified a default Cypher version, it is possible to set a different Cypher version for individual queries.
+For example, if a database is created with Cypher 5 as its default language, prepending a query run on that database with `CYPHER 25` ensures the query uses Cypher 25.
+
+[[procedures-and-functions]]
+=== Procedures and functions
+
+link:{neo4j-docs-base-uri}/operations-manual/current/procedures[Procedures] and xref:functions/index.adoc[functions] (including built-in and link:{neo4j-docs-base-uri}/apoc/current/[APOC]) are tied to a specific Cypher language version.
+Therefore, procedures and functions in Neo4j 2025.03+ and APOC 2025.03+ (both of which have Cypher 25 as their default language), may behave differently depending on what version of Cypher is used.
+
+For example, APOC 2025.03 removed Cypher 25 support of the procedure `apoc.create.uuids()`, meaning it is not available to queries running Cypher 25.
+However, it can still be used on APOC 2025.03 if queries are prepended with `CYPHER 5`, or if the database’s default version is set to `CYPHER 5`.
+In this case, Neo4j will use APOC and Cypher as they existed at the time of the 5.26 long-term support (LTS) release.
+
+.Using a procedure removed in Cypher 25 with APOC 2025.03+
+[source, cypher]
+----
+CYPHER 5
+CALL apoc.create.uuids(10)
+----
+
+[[cypher-selection-with-other-query-options]]
+=== Combine Cypher version selection with other query options
+
+It is possible to combine Cypher version selection with other xref:planning-and-tuning/query-tuning.adoc[query options].
+The below example selects both the version and the xref:planning-and-tuning/runtimes/concepts.adoc[runtime] of Cypher for the same query:
+
+.Combining Cypher version selection with other query options
+[source, cypher]
+----
+CYPHER 5 runtime=parallel
+MATCH (n:Person)
+RETURN n.name
+----
+
+[[select-default-cypher-version]]
+== Select the default Cypher version when creating a database
+
+Databases created on Neo4j 2025.03 or later will have Cypher 25 as their default language.
+This is true for the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/#manage-database-systems[system, standard, and composite] Neo4j databases.
+
+To select a different default Cypher version for a database, add `DEFAULT LANGUAGE <language version>` to the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/create-databases/[`CREATE DATABASE`] statement.
+This can be done on any Neo4j database.
+
+[NOTE]
+Selecting a default Cypher version on a database requires the `SET DEFAULT LANGUAGE` privilege.
+
+Selecting `CYPHER 5` as the default database language ensures that every query run on that database uses the language as it existed at the time of the Neo4j 5.26 long-term support release.
```

Comment:
do we want to mention here something along the lines of `unless overriden in the query`?